tests/test_checks.py

```diff
@@ -0,0 +1,142 @@
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+from django.core import checks
+
+from corsheaders.conf import corsheaders_settings
+from corsheaders.checks import check_settings
+
+from corsheaders.defaults import default_headers, default_methods
+
+class SettingsTests(SimpleTestCase):
+
```

Comment:
I'm not sure how to do this.

```
    @override_settings(ROOT_URLCONF='.')
    def test_call_command_check(self):
        try:
            management.call_command('check')
        except Exception as err:
            assert isinstance(err, SystemCheckError)
```

I think I need to change system settings in `check_settings` function and expect `SystemCheckError` in `test_call_command_check`?